moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -179,14 +205,37 @@ const ProjectWorkActivitiesTable = () => {
     setEditActivity,
   });
 
+  /**
+   * Initialize which columns should be visibile - must be memoized to safely
+   * be used with useHiddenColumnsSettings hook
+   */
+  const defaultHiddenColumnSettings = useMemo(
```

Comment:
Cool, so this bridges to the existing code and default column visibility to the DataGrid's `columnVisibilityModel`. 😎